app/src/models/Card.py

```diff
@@ -3,15 +3,17 @@
 
 class Card:
 
-    def __init__(self, card_id: int, deck_id: int, word: str, translation: str):
-        self.id = card_id
+    def __init__(self, card_id: int, deck_id: int, word: str, translation: str, created_at: str, updated_at: str):
+        self.card_id = card_id
```

Comment:
why card_id? you are inside Card class, id is enough

Reply:
id is function in python so it's better not use it as a name

Reply:
done